src/pretix/presale/templates/pretixpresale/waiting.html

```diff
@@ -21,6 +21,21 @@
 
         <h1>{% trans "We are processing your request …" %}</h1>
 
+        {% if percentage %}
+            <div class="progress">
+                <div class="progress-bar progress-bar-success progress-bar-{{ percentage|floatformat:0 }}">
+                </div>
+            </div>
+            {% if steps %}
+                <p class="steps">
+                    {% for step in steps %}
+                        <span class="fa fa-fw {% if step.done %}fa-check text-success{% else %}fa-cog fa-spin text-muted{% endif %}"></span>
+                        {{ step.label }}<br>
+                    {% endfor %}
+                </p>
```

Comment:
Regarding a11y, I would make this an `<ol>`, something like this:

```suggestion
                <ol class="steps">
                    {% for step in steps %}
                        <li><span class="fa fa-fw {% if step.done %}fa-check text-success{% else %}fa-cog fa-spin text-muted{% endif %}"></span>
                        {{ step.label }}<span class="sr-only">, {% if step.done %}{% trans "done" %}{% else %}{% trans "in progress" %}{% endif %}"></span></li>
                    {% endfor %}
                </ol>
```